.gitignore

```diff
@@ -132,11 +132,11 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
-TripPlanner - Project/secret.json
-/TripPlanner - Project/APIsApp/api_keys.json
+backend/secret.json
```

Comment:
maybe instead of using `backend` and `frontend`, could we use something like `api` for backend and `app` for frontend? I believe it's more correct semantically

Reply:
I don't have strong opinions about this though

Reply:
I see, the problem is that in our backend we already have app called api.

Reply:
np, let's just stick with current structure for now, as it isn't necessarily wrong